config/fields/structure.php

```diff
@@ -181,11 +181,22 @@
 		},
 	],
 	'save' => function ($value) {
-		$data = [];
-		$form = $this->form();
-
-		foreach ($value as $row) {
-			$row = $form->reset()->submit(input: $row, passthrough: true)->toStoredValues();
+		$data     = [];
+		$form     = $this->form();
+		$defaults = $form->defaults();
+
+		foreach ($value as $index => $row) {
+			$row = $form
+				->reset()
+				->fill(
+					input: $defaults,
+					passthrough: true
+				)
+				->submit(
+					input: $row,
+					passthrough: true
```

Comment:
Question somewhat unrelated to this PR: Why do we use `passthrough` for the `submit()` call here? Doesn't it allow to pass unfiltered and unsanitized data for extra fields from the frontend?